chain/events/state/predicates_test.go

```diff
@@ -36,39 +34,6 @@ func init() {
 	dummyCid, _ = cid.Parse("bafkqaaa")
 }
 
-type mockAPI struct {
```

Comment:
Note: There are no logic changes in the predicate tests, I just moved some things into a shared directory so I could use them from the deal state matcher tests